cmd/kube-apiserver/app/server.go

```diff
@@ -195,7 +195,7 @@ func CreateServerChain(completedOptions completedServerRunOptions, stopCh <-chan
 	if err != nil {
 		return nil, err
 	}
-	aggregatorServer, err := createAggregatorServer(aggregatorConfig, kubeAPIServer.GenericAPIServer, apiExtensionsServer.Informers)
+	aggregatorServer, err := createAggregatorServer(aggregatorConfig, kubeAPIServer.GenericAPIServer, apiExtensionsServer.Informers, kubeAPIServerConfig.GenericConfig.EgressSelector)
```

Comment:
RE. Daniel's first point, I think we don't need to change the signature here, the egressSelector is in aggregatorConfig.GenericConfig.Config.EgressSelector already.